release/downloadIstioCandidate.sh

```diff
@@ -103,7 +103,11 @@ without_arch() {
 }
 
 # Istio 1.6 and above support arch
-ARCH_SUPPORTED=$(echo "$ISTIO_VERSION" | awk  '{ ARCH_SUPPORTED=substr($0, 1, 3); print ARCH_SUPPORTED; }' )
+ARCH_SUPPORTED=$(echo "$ISTIO_VERSION" | awk  '{ ARCH_SUPPORTED=substr($0, 1, 4); print ARCH_SUPPORTED; }' )
```

Comment:
I haven't looked, but can't we just print the first two fields (break on a `.`) separated by a period. Seems to be better that assuming some number of characters.

Reply:
Thanks